.github/ISSUE_TEMPLATE/sub_rc_checklist.md

```diff
@@ -18,8 +18,7 @@ labels: task
 * [ ] Build and upload conda packages on buildfarm (check `upload`).
 * [ ] Verify packages uploaded to Anaconda Cloud and move to
   `numba/label/main`.
-* [ ] Build wheels (`$PYTHON_VERSIONS`) on the buildfarm.
-* [ ] Build sdist locally using `python setup.py sdist --owner=ci --group=numba` with umask `0022`.
+* [ ] Build wheels and sdist on the buildfarm (check "upload").
```

Comment:
Suggest adding this:
```
* [ ] Commit build farm configuration and tag with release version.
```
somewhere above before any "building" steps are mentioned.

Reply:
I think it is generally taking a snapshot of the what was used to build the packages is probably a good idea. Though I would propose to add this step to `post-release` steps. In my experience, the farm will often need changes, even after the tag was made. I think that tagging the config version that was used to perform the final buildout is probably closest to what we want.